glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/minishift/minishift
 import:
 - package: github.com/docker/machine
-  version: 91e368eb7422665278b1d5665139ee0f9980b588
+  version: 15fd4c70403bab784d91031af02d9e169ce66412
```

Comment:
I'd prefer doing it now. It seems connected really. You also should not have to clear the caches completely. Once everything works maybe for good measure. 

For me it does not take so long to regenerate. I can otherwise try tomorrow. I'd really like to get way from depending on sha's like this